app/human_svc.py

```diff
@@ -42,12 +42,14 @@ async def load_available_workflows(self):
     """ PRIVATE """
 
     async def _load_workflow_module(self, root, workflow_file):
-        module = os.path.join(root, workflow_file.split('.')[0]).replace(os.path.sep, '.')
+        module_path = os.path.join(root, workflow_file.split('.')[0]).replace(os.path.sep, '.')
         try:
-            loaded = getattr(import_module(module), 'load')(driver=None)
-            await self.data_svc.store(Workflow(name=loaded.name, description=loaded.description, file=workflow_file))
+            module = import_module(module_path)
+            workflow_name = getattr(module, 'WORKFLOW_NAME')
+            workflow_description = getattr(module, 'WORKFLOW_DESCRIPTION')
+            await self.data_svc.store(Workflow(name=workflow_name, description=workflow_description, file=workflow_file))
         except Exception as e:
-            self.log.error('Error loading extension=%s, %s' % (module, e))
+            self.log.error('Error loading extension=%s, %s' % (module_path, e))
```

Comment:
Tested this in my Ubuntu VM this morning and got the errors below when standing up the CALDERA server. Adding `webdriver_manager` to the requirements.txt should resolve, but did this come up for anyone else? 

```
2022-01-12 08:52:33 - ERROR (human_svc.py:52 _load_workflow_module) Error loading extension=plugins.human.pyhuman.app.workflows.browse_web, No module named 'webdriver_manager'
2022-01-12 08:52:33 - ERROR (human_svc.py:52 _load_workflow_module) Error loading extension=plugins.human.pyhuman.app.workflows.google_search, No module named 'webdriver_manager'
```